renovate.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "local>elastic/renovate-config"
```

Comment:
```suggestion
    "local>elastic/renovate-config",
    "github>elastic/renovate-config:only-chainguard"
```

Reply:
I'd leave the config as is for now and added a config to make all our test/compile-only dependencies opt-in via the dependency dashboard.

Should we also maybe remove the version pinning for GH-actions?

Reply:
If that's what the team wanna use these days, that's totally fine. 

I'm not familiar with using renovate for GH actions. IMO I like dependabot for managing GH Actions, since we can control the secret access and it's what we use in the remaining repositories and we control the marvellous Mondays when we review all the dependencies for all our projects.

However, if the Java team prefers to use Renovate, we can discuss how to do this in a separate follow-up.